scripts/auto-cpufreq-install.sh

```diff
@@ -70,29 +70,25 @@ elif [ "$(ps h -o comm 1)" = "systemd" ];then
     systemctl enable auto-cpufreq
 # Install script for openrc / sysvinit
 elif [ "$(ps h -o comm 1)" = "init" ];then
-   mkdir /etc/inittab
+   ls /etc/inittab
     if [ -e "/etc/inittab" ]; then 
         echo -e "\n* Deploy auto-cpufreq sysvinit unit file"
-        cp /usr/local/share/auto-cpufreq/scripts/auto-cpufreq-openrc /etc/init.d/auto-cpufreq
-	
-        echo -e "\n* "Reloading sysvinit manager configuration"
-	service auto-cpufreq reload
+        cp /usr/local/share/auto-cpufreq/scripts/auto-cpufreq-sysvinit /etc/init.d/auto-cpufreq
  
```

Comment:
I think you need a 
`chmod +x /etc/init.d/auto-cpufreq`
here
The copying to /etc/init.d works on my system but the script is not executable.